cirq-google/cirq_google/engine/engine_processor_test.py

```diff
@@ -301,6 +327,39 @@ def test_get_missing_device():
         _ = processor.get_device()
 
 
+def test_get_sampler_initializes_default_device_configuration() -> None:
+    processor = cg.EngineProcessor(
+        'a',
+        'p',
+        EngineContext(),
+        _processor=quantum.QuantumProcessor(
+            default_device_config_key=quantum.DeviceConfigKey(
+                run="run", config_alias="config_alias"
+            )
+        ),
+    )
+    sampler = processor.get_sampler()
+
+    assert sampler.run_name == "run"
+    assert sampler.device_config_name == "config_alias"
+
+
+def test_get_sampler_loads_processor_with_default_device_configuration() -> None:
+    client = FakeEngineClient()
```

Comment:
This looks almost the same as a mock, which would skip creating the `FakeEngineClient` class; wdyt? 
```py
client = mock.Mock(engine_client.EngineClient)
client.get_processor.return_value = quantum.QuantumProcessor(
   ...
)
```

Reply:
+1, added